Python/cp/staircase_problem.py

```diff
@@ -0,0 +1,27 @@
+## This program finds the total number of possible combinations that can be used to
+## climb statirs . EG : for 3 stairs ,combination and output will be 1,1,1 , 1,2 , 2,1 i.e 3 . 
+
+def counting_stairs(stair_number):
+    result = stair_number
+    ##Using Recursion to find the total number of stairs as f(a) = f(a-1) + a(a-2) 

```

Comment:
I guess `# Python is a Cool Language` and `## Python is a Cool Language` would work the same. Why the use of `##` to enclose comments?